cpp/email_watcher.cc

```diff
@@ -198,21 +189,62 @@ std::map<std::string, time_t> LoadSectionNamesToLastSeenTimeWindowsMap() {
     return section_names_to_last_seen_time_map;
 }
 
-
+#if 0
```

Comment:
the `#if 0` section should be removed before merging

Reply:
No because I am still working on it.  I had to adapt it to the new API o/w the submitted code wouldn't compile.